### PR TITLE
Shipment.toString() added

### PR DESCRIPTION
**Pull request by michalmac**

### CODE REVIEW

File: jsprit-core/src/main/java/com/graphhopper/jsprit/core/problem/job/Shipment.java [hunk 1]
```diff
@@ -448,6 +448,21 @@ public TimeWindow getPickupTimeWindow() {
...
+     */
+    @Override
+    public String toString() {
+        return "[id=" + id + "][name=" + name + "][pickupLocation=" + pickupLocation_
+                + "][deliveryLocation=" + deliveryLocation_ + "][capacity=" + capacity
+                + "][pickupServiceTime=" + pickupServiceTime + "][deliveryServiceTime="
+                + deliveryServiceTime + "][pickupTimeWindow=" + pickupTimeWindow
+                + "][deliveryTimeWindow=" + deliveryTimeWindow + "]";
+    }
+
```
Comment (oblonski) [inline jsprit-core/src/main/java/com/graphhopper/jsprit/core/problem/job/Shipment.java]: @michalmac Thanks. There might be more than one pickup and delivery time window. Would you mind to account for this also? For example, [pickupTimeWindow1=][pickupTimeWindow2=] or [pickupTimeWindow_1 ...]?

Comment (oblonski) [reply jsprit-core/src/main/java/com/graphhopper/jsprit/core/problem/job/Shipment.java]: or like this "[pickupTimeWindows=[timeWindow=" + pickupTimeWindow[0] + "][...]]“

**Comment by oblonski** [comment]
Perfect! Thanks.

File: jsprit-core/src/main/java/com/graphhopper/jsprit/core/problem/job/Shipment.java (hunk 1 above)
Comment (michalmac) [reply jsprit-core/src/main/java/com/graphhopper/jsprit/core/problem/job/Shipment.java]: Sure, my change was according to what was in Service. But now also Service.toString() prints out all time windows.

BTW. I cleaned up slightly both classes. There are still 2 things that could be done:
- get rid of timeWindow from ServiceBuilder and pickup/deliveryTimeWindow from ShipmentBuilder -- they are unused
- we could get rid of some generics-related warnings by changing XXX.Builder.newInstance() to XXX.Builder.newXXXInstance in Service/Break/Delivery/Pickup... but this is a change to API, so not sure if we want it

**Comment by michalmac** [comment]
@oblonski, not sure if you have seen this (in the pull request review discussion), so I am re-sending it here:
There are still 2 things that could be done:
- get rid of timeWindow from ServiceBuilder and pickup/deliveryTimeWindow from ShipmentBuilder -- they are unused
- we could get rid of some generics-related warnings by changing XXX.Builder.newInstance() to XXX.Builder.newXXXInstance in Service/Break/Delivery/Pickup... but this is a change to API, so not sure if we want it

**Comment by oblonski** [comment]
@michalmac if it is unused and we dont break the API, we can remove it. When it comes to newXXXInstance() I d leave it as it is for now and if we want to change it, mark it as decrecated first and change in a later version.

**Comment by michalmac** [comment]
thanks @oblonski, so the first issue is addressed with PR #384

**Pull request merged**